DOCS/man/input.rst

```diff
@@ -555,8 +555,8 @@ Remember to quote string arguments in input.conf (see `Flat command syntax`_).
 
     ``passthrough_stdin`` (``MPV_FORMAT_FLAG``)
         If enabled, wire the new process' stdin to mpv's stdin (default: no).
-        Before mpv 0.33.0, this argument did not exist, but the default was if
-        it was set to ``yes``.
```

Comment:
formatting?

Reply:
I still don't understand. :/ Do you want me to format added lines in a different way? I used the default vim "gq". I don't know what others used.